deploy/helm/sumologic/values.yaml

```diff
@@ -2464,8 +2464,7 @@ opentelemetry-operator:
 
   ## Specific for OpenTelemetry Operator chart values
   admissionWebhooks:
-    failurePolicy: Fail
-    enabled: true
+    create: true
```

Comment:
will this change affect helm chart upgrades in anyway?

Reply:
I don't think so. If someone disabled webhooks, they'll get an error and will need to set `create: false` instead of `create: true`. The other values I removed because we were using defaults anyway.

Reply:
> changes to CRD handling that will require changes on our end as well.
ok, what about the upcoming changes mentioned here. Is there a breaking changes in the operator that we should be tracking to upgrade to the next version?